sigs/cncf-sigs.md

```diff
@@ -111,8 +111,23 @@ As a starting point let’s be inspired by CNCF OSS Projects and by K8s SIGs.  T
 
 #### Tech Lead
 
-* Leads projects in the SIG’s area.
-* Has the time and ability to perform deep technical dives on projects.   Projects may include formal CNCF projects or other projects in the area covered by the SIG.
+A Tech Lead will typically have a track record of participation in SIG
+activities and/or as TOC Contributors; however, this is not a requirement,
```

Comment:
"as" does not work after "and/or"

Reply:
why not?  a tech lead could have a track record 
*  of participation in SIG
* as a TOC Contributor

I realize there's a mis-match in singular "A Tech Lead" / plural "TOC Contributors"... but I agree the whole sentence is a bit awkward.... will try to reword this.  Suggestions welcome!